source/uwp/Renderer/idl/AdaptiveCards.XamlCardRenderer.idl

```diff
@@ -777,6 +793,7 @@ namespace AdaptiveCards
         interface IAdaptiveActionEventArgs : IInspectable
         {
             [propget] HRESULT Action([out, retval] IAdaptiveActionElement** action);
+            [propget] HRESULT Inputs([out, retval] HSTRING* value);
```

Comment:
@matthidinger We're returning a string here for the input values? Why not a KeyValuePair of string/string?